src/Layouts/stars/feed/starDesc.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  TextField,
+  Typography,
+  FormControl,
+  InputLabel,
+  Select,
+  Chip,
+  MenuItem,
+  Input,
+} from "@mui/material";
+import { starType, resources } from "../../../assets/star";
+import { useState } from "react";
+
+interface starProps {
+  star: starType;
+}
+
+const StarDesc = ({ star }: starProps) => {
+  const [resourceList, setResourceList] = useState(star.resources);
+
+  return (
+    <div className="starDesc">
+      <div className="header">
+        <h1>
+          <span
+            id="priority"
+            style={{
+              color:
+                star.severity === 1
+                  ? "red"
+                  : star.severity === 2
+                  ? "orange"
+                  : star.severity === 3
+                  ? "yellow"
+                  : "green",
+            }}
+          >
+            {star.priority > 0 ? star.priority : ""}{" "}
```

Comment:
do `&&` instead  of `: ? ` where the else value is not important
 
**but not on the retrun of components ** there you still  need `cond? <comp>: null` (otherwise  you will retrun Boolean instead  of component

Reply:
why would this work? 
the value can be set to 0 but it always exists, therefore && will return true also for 0, wont it?